scripts/test/large.py

```diff
@@ -0,0 +1,78 @@
+from modules import utils
+from modules.logger import log
+from unittest import TextTestRunner
+
+import sys
+import unittest
+
+
+class PsutilCollectorLargeTest(unittest.TestCase):
+
+    def setUp(self):
+        # set and download required binaries (snapd, snapctl, plugins)
+        self.binaries = utils.set_binaries()
+        utils.download_binaries(self.binaries)
+
+        log.debug("Starting snapd")
+        self.binaries.snapd.start()
+        if not self.binaries.snapd.isAlive():
+            self.fail("snapd thread died")
+
+        log.debug("Waiting for snapd to finish starting")
+        if not self.binaries.snapd.wait():
+            self.binaries.snapd.kill()
+            self.fail("snapd not ready, timeout!")
+
+    def test_psutil_collector_plugin(self):
+        # load psutil collector
+        loaded = self.binaries.snapctl.load_plugin("snap-plugin-collector-psutil")
+        self.assertTrue(loaded, "psutil collector loaded")
+
+        # check available metrics, plugins and tasks
+        metrics = self.binaries.snapctl.list_metrics()
+        plugins = self.binaries.snapctl.list_plugins()
+        tasks = self.binaries.snapctl.list_tasks()
+        self.assertGreater(metrics, 0, "Metrics available {} expected {}".format(metrics, 0))
```

Comment:
Perhaps PEP-8 division will kill me for that comment but for such sort format strings you can use "%d xxx %d" % (metrics, 0)